src/usingModules.js

```diff
@@ -0,0 +1,15 @@
+/*
+import _ from 'lodash';
+import todo from '../modules/todo'
+
+const description = document.querySelector('#desc-input').value;
+const counter = -1;
+
+const AddToDo = (description) => {
+  const isCompleted = 'false';
+  const index = counter+1;
+  counter++;
+const newToDo = new todo(description, isCompleted, index);
+list.push(newToDo);
+}
+*/
```

Comment:
- [ ] Removing commented code not only makes it harder to find later, but it also makes it so people in the future don't know it existed before. For finding the code, there are [tools and git commands](http://stackoverflow.com/q/278192/971592) to help you look at the history of a file. Please remove all commented code and unnecessary files from your code base :smile:

Reply:
Thank you @codecaiine for sharing this link!